package/doc/sphinx/source/documentation_pages/pickle_universe.rst

```diff
@@ -0,0 +1,112 @@
+.. Contains the formatted docstrings for the serialization of universe located 
```

Comment:
Put this as under [doc/sphinx/source/documentation_pages/coordinates](https://github.com/MDAnalysis/mdanalysis/tree/develop/package/doc/sphinx/source/documentation_pages/coordinates) and add an entry to the second TOC (for developers) in [doc/sphinx/source/documentation_pages/coordinates_modules.rst](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/doc/sphinx/source/documentation_pages/coordinates_modules.rst):
```reST
The remaining pages are primarily of interest to developers. Programmers and anyone trying to implement new functionality should first read the :ref:`Trajectory API` and then look at the following pages. 

.. toctree::
   :maxdepth: 1

   coordinates/base
   coordinates/core
   coordinates/pickle_universe.rst
   coordinates/chain
   coordinates/XDR

In particular, all trajectory readers have to be :ref:`serializable<serialization>` and they should pass all tests
available in the ``MDAnalysisTests.coordinates.base.MultiframeReaderTest`` or ``MDAnalysisTests.coordinates.base.BaseReaderTest``.
```